com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs

```diff
@@ -581,13 +582,13 @@ static void Drawer_SectionDynamicResolutionSettings(SerializedHDRenderPipelineAs
             --EditorGUI.indentLevel;
 
 #if ENABLE_NVIDIA && !ENABLE_NVIDIA_MODULE
-            EditorGUILayout.BeginHorizontal();
-            EditorGUILayout.HelpBox(Styles.DLSSPackageLabel, MessageType.Info);
-            if (GUILayout.Button(Styles.DLSSInstallButton, GUILayout.ExpandWidth(true), GUILayout.ExpandHeight(true)))
+            if (s_DisplayNvidiaModuleButtonInstall)
```

Comment:
We shouldn't check if the `com.unity.modules.nvidia` is being already added into the `PackageManager`?

Reply:
1) When the nvidia package is already in the manager the preprocessor directive ENABLE_NVIDIA_MODULE is true, so this UI doesnt get painted at all. Is through pre processor directives that we can see if a moudle is included or not. See the asmdef of this package, thats where you map the preprocessor directives.

2) The global s_DisplayNVIDIAModuleButtonInstall is part of a bug, there is a 3 second delay when the button gets pressed and until HDRP gets recompiled. During this time if the user spams click then we get a failure.